presto-pinot/src/test/java/io/prestosql/pinot/TestingPinotCluster.java

```diff
@@ -64,29 +65,29 @@
     public TestingPinotCluster()
     {
         httpClient = new JettyHttpClient();
-        zookeeper = new GenericContainer<>("zookeeper:3.5.6")
+        zookeeper = new GenericContainer<>(parse("zookeeper:3.5.6"))
```

Comment:
I'd advise against imporing `parse` statically, since it doesn't look unambiguous in the code.